src/App.tsx

```diff
@@ -1,13 +1,15 @@
 import { useState } from "react";
 import "./App.css";
-import { Button } from "Components";
+import { Button } from "./components";
```

Comment:
same here
This is relative path. Need to use absolute path
This change is not needed

Reply:
If I don't make these changes , code is showing errors in my IDE.